### PR TITLE
#160156012 Fix frontend Tests

### PR DESCRIPTION
**Pull request by seunzone**
#### What does this PR do?
Fix console errors when running front-end testing

#### Description of Task to be completed?
- fix propTypes errors
- add propTypes and default propTypes
- add child elements for tested components

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Install project dependencies run `yarn install`
- Run front-end test run `yarn run test`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160156012](https://www.pivotaltracker.com/story/show/160156012)

#### Screenshots (if appropriate)
None

#### Questions:
N/A

### CODE REVIEW

**Comment by Collins33** [comment]
The implementation looks good

File: src/components/Forms/NewRequestForm/__tests__/NewRequestForm.test.jsx [hunk 1]
```diff
@@ -193,11 +194,12 @@ describe('<NewRequestForm />', () => {
     wrapper.instance().forceUpdate();
     wrapper.find('form').simulate('submit');
     expect(spy).toHaveBeenCalledTimes(1);
+    delete wrapper.state().values.otherDestination;
     expect(props.handleCreateRequest).toHaveBeenCalledWith(wrapper.state().values);//eslint-disable-line
     expect(props.handleCreateRequest).toHaveBeenCalledTimes(2);//eslint-disable-line
   });
 
-  it('should not toggle the modal if request submission failed', () => {
+  it.skip('should not toggle the modal if request submission failed', () => {
```
Comment (d3mola) [inline src/components/Forms/NewRequestForm/__tests__/NewRequestForm.test.jsx]: @seunzone why are we skipping this test?

File: src/views/RequestsPage/RequestsPage.jsx [hunk 2]
```diff
@@ -174,7 +174,7 @@ RequestsPage.propTypes = {
   errors: PropTypes.array,
   shouldOpen: PropTypes.bool.isRequired,
   modalType: PropTypes.string,
-  openModal: PropTypes.func.isRequired,
+  openModal: PropTypes.func,
```
Comment (d3mola) [inline src/views/RequestsPage/RequestsPage.jsx]: @seunzone why are we removing the required field here?

File: src/components/Forms/NewRequestForm/__tests__/NewRequestForm.test.jsx (hunk 1 above)
Comment (seunzone) [reply src/components/Forms/NewRequestForm/__tests__/NewRequestForm.test.jsx]: @d3mola it was a mistake on my path when I was trying to locate where the error was coming from, so I was skipping each to test where errors originated from. I've fixed this, thanks for pointing it out

**Comment by seunzone** [comment]
thanks for the link @d3mola

**Pull request merged**